storybookjs/storybook
# Update global hook for Vue Devtools

Issue:

## What I did

Right now, Vue Devtools cannot identify Vue root nodes within iframes. We can link a global hook from the parent window to the iframe window and devtools will successfully recognize that the page includes a Vue application. This causes the devtools add the tab to browser debugging windows. 

Secondly, adding a `__VUE_DEVTOOLS_CONTEXT__` variable and defining it as the iframe document, allows the devtools to properly walk the context and identify components. To work, this requires a change to Vue Devtools as well.

## How to test

Run storybook and attempt to use Vue Devtools. Without the corresponding PR to add `__VUE_DEVTOOLS_CONTEXT__`, you should still see Vue Devtools but Components will not appear correctly

I'm going to submit a PR to devtools which I'll reference here as well.
